sal_interfaces/MTM2/MTM2_Events.xml

```diff
@@ -101,4 +101,32 @@
     </item>
 </SALEvent>
 
+<SALEvent>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_logevent_isForceBalanceSystemOn</EFDB_Topic>
+    <Alias>isForceBalanceSystemOn</Alias>
+    <Description>Force balance (FB) system is on or off?</Description>
+    <item>
+        <EFDB_Name>status</EFDB_Name>
```

Comment:
What about just naming this `on`? and it could be True/False?

Reply:
Done in commit: 9c5cc3f. Thanks!